setup.cfg

```diff
@@ -1,5 +1,13 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
Viper is ready to use right now, not in production environment but if in test environment. So, VIper has solid features :) At least compile a whole contract.

But it's ok start from 0.0.1 :+1: